renovate.json

```diff
@@ -3,5 +3,12 @@
   "extends": ["github>konflux-ci/mintmaker//config/renovate/renovate.json"],
   "timezone": "America/New_York",
   "schedule": ["before 10pm"],
-  "enabledManagers": ["asdf", "tekton"]
+  "enabledManagers": ["asdf", "tekton", "github-actions"],
+  "github-actions": {
+    "enabled": true,
+    "schedule": ["before 10pm"],
```

Comment:
Won't this increase the frequency of the github-actions updates from weekly to daily?

Reply:
yes, but it should group them and that will help me while I try and sort through this. I can make a note in the Jira to move it back to weekly if we prefer that.